tests/nosetests/dd_tests/dd_test.py

```diff
@@ -10,7 +10,9 @@
 
 from contextlib import contextmanager
 from collections import namedtuple
-from rpmfluff import SourceFile, GeneratedSourceFile, SimpleRpmBuild, expectedArch, make_elf
+from rpmfluff import SourceFile, GeneratedSourceFile, SimpleRpmBuild
+from rpmfluff.utils import expectedArch
+from rpmfluff.make import make_elf
```

Comment:
Please remove this. We will pin old version for RHEL-8.

Reply:
OK, I'll drop it before merging. :)